CHANGELOG.md

```diff
@@ -15,19 +35,19 @@ This project adheres to [Semantic Versioning](http://semver.org/).
 ### Added
 
 - Implement Resin API Key. User can authorize by credentials, auth token or set Resin API key as RESIN_API_KEY environment variable.
-- Implement `resin.models.device.restart()`
-- Implement `resin.models.device.has_device_url()`
-- Implement `resin.models.device.get_device_url()`
-- Implement `resin.models.device.enable_device_url()`
-- Implement `resin.models.device.disable_device_url()`
+- Implement `resin.models.device.Device.restart()`
+- Implement `resin.models.device.Device.has_device_url()`
+- Implement `resin.models.device.Device.get_device_url()`
+- Implement `resin.models.device.Device.enable_device_url()`
+- Implement `resin.models.device.Device.disable_device_url()`
```

Comment:
You should document these changes in a section "Changed" for the current release, not by editing the previous changelog! Same goes for the other edits below.